features/git-town-ship/current_branch/on_feature_branch/without_open_changes/delete_remote_branch_disabled.feature

```diff
@@ -0,0 +1,58 @@
+Feature: Skip deleting the remote branch when shipping the current branch
```

Comment:
These tests don't really make sense to me as this option only really works when shipping through the github API right? Otherwise the PR wouldn't be merged and thus the branch not auto deleted?

Reply:
That said, I'm not certain how we would test this otherwise.

Reply:
Good point! After thinking about this for a while, I realized that the only way to resolve this is to build this the right way 😄  It is not up to Git Town to have knowledge or opinions on whether the remote server deletes the branch or not. GitHub isn't the only remote that does such things. There might be other types of remotes who do something with shipped branches. One could build this into any Git server via a post-merge hook or bot. All that Git Town can do here is provide a setting to not delete remote branches, and apply it consistently. 

This results in correct behavior in all currently known circumstances:
- when GitHub config is set up, and we are ONLINE, it should use this setting
- when GitHub config is set up, and we are OFFLINE, it never deletes the branch
- when GitHub config is NOT set up, and we are ONLINE, it should use this setting
- when GitHub config is NOT set up, and we are OFFLINE, it never deletes the branch

If the user doesn't like any of this behavior, then they can change the configuration.

I have updated the documentation to make it clear that Git Town always applies this setting.